sapi/litespeed/lsapilib.c

```diff
@@ -2672,7 +2672,11 @@ int LSAPI_ParseSockAddr( const char * pBind, struct sockaddr * pAddr )
             ((struct sockaddr_in *)pAddr)->sin_addr.s_addr = htonl( INADDR_LOOPBACK );
         else
         {
+#ifdef HAVE_INET_PTON
+            inet_pton(AF_INET, p, &((struct sockaddr_in *)pAddr)->sin_addr);
+#else
             ((struct sockaddr_in *)pAddr)->sin_addr.s_addr = inet_addr( p );
+#endif
             if ( ((struct sockaddr_in *)pAddr)->sin_addr.s_addr == INADDR_BROADCAST)
```

Comment:
I was confused by this code at first ... apparently INADDR_BROADCAST is really the same as INADDR_NONE, so this is also just an error check, and should be handled the same way you did for the other occurrence.

Reply:
Same for me...
Fixed